langchain_tools_demo/main.py

```diff
@@ -63,13 +65,34 @@ def index(request: Request):
     )
 
 
+@app.post("/login/google")
+async def login_google(
+    request: Request,
+):
+    request.session.clear()
+    form_data = await request.form()
+    user_id_token = str(form_data.get("credential", ""))
```

Comment:
This needs to be a required parameter

Reply:
Deleted default value and added check if credential exists